src/gui/mzroll/pollyelmaveninterface.cpp

```diff
@@ -13,6 +13,7 @@
 #include "pollywaitdialog.h"
 #include "projectdockwidget.h"
 #include "tabledockwidget.h"
+#include "peakdetectiondialog.h"
```

Comment:
This is no longer needed here now that we have moved the slot connection to login-form, right?